tools/report-converter/tests/unit/output/html/plist_to_html_test.py

```diff
@@ -157,3 +160,16 @@ def test_html_builder(self):
         self.__test_html_builder('notes')
         self.__test_html_builder('macros')
         self.__test_html_builder('simple')
+
+    def test_html_for_inclusion(self):
+        """ Test generating html files for header inclusions. """
+        output_dir = self.__test_html_builder('inclusion')
+        index_html = os.path.join(output_dir, "index.html")
+
+        report_count = 0
+        with open(index_html, 'r', encoding="utf-8", errors="ignore") as f:
+            for line in f:
+                if re.search("<td file=", line):
+                    report_count += 1
+
+        self.assertEqual(report_count, 4)
```

Comment:
This became **4** now, which is also an even number. :slightly_frowning_face: 
The idea was that only 1 out of the 2 TUs will have a `return X;`, the other won't, so we get (for the project) 1 *"non-void function does not return a value"* warning. Previously we had 2, now we have 0, the parity is the same, so we still can't catch a "duplication" mistake.

Reply:
@whisperity Fixed.